docs/src/main/sphinx/eventlistener/http-event-listener.rst

```diff
@@ -0,0 +1,92 @@
+===================
+Http Event Listener
+===================
+
+The Http Event Listener plugin allows streaming of query events, encoded in
+JSON format, to an external service for further processing, by POSTing them
+to a specified URI.
+
+Rationale
+---------
+
+This event listener is a simple first step into better understanding the usage
+of a datalake using query events provided by Trino. These can provide CPU and memory
+usage metrics, what data is being accessed (with resolution down to specific columns),
+and meta-information about the query.
+
+Running this system decoupled from Trino means there will be no performance hit and no
```

Comment:
Running the capture system separate from Trino reduces the performance impact and avoids downtime for non-client-facing changes.